src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -126,7 +186,8 @@ private void createTopic(final String topic, final int partitionsNum, final int
                 final Properties topicConfig = new Properties();
                 topicConfig.setProperty("retention.ms", Long.toString(retentionMs));
                 topicConfig.setProperty("segment.ms", Long.toString(rotationMs));
-                AdminUtils.createTopic(zkUtils, topic, partitionsNum, replicaFactor, topicConfig);
+                AdminUtils.createTopic(zkUtils, topic, partitionsNum, replicaFactor, topicConfig,
+                        RackAwareMode.Enforced$.MODULE$);
```

Comment:
Ok, so what I got is that once we add rack awareness, by specifying the `broker.rack` property in `server.properties` this is going to be immediately used by Nakadi when creating new topics. So sounds like a good thing. It's just not clear if it has any difference right now, since we do not have `broker.rack` configured.